Don't check whether gnuplot is present in $PATH, because we know it is.

diff --git a/eazy-gnuplot.asd b/eazy-gnuplot.asd
index 7b1c09b..4657cd6 100644
--- a/eazy-gnuplot.asd
+++ b/eazy-gnuplot.asd
@@ -37,7 +37,4 @@
                                :fill-pointer t)))
           (setf (fill-pointer seq) (read-sequence seq stream))
           seq)))
-  :perform (load-op :before (op c)
-                    (unless (zerop (nth-value 2 (uiop:run-program "which gnuplot" :ignore-error-status t)))
-                      (warn "Could not find GNUPLOT in $PATH")))
   :in-order-to ((test-op (test-op eazy-gnuplot.test))))
